package.json

```diff
@@ -49,15 +51,10 @@
     "@testing-library/react-hooks": "^3.2.1",
     "babel-eslint": "^10.0.1",
     "cross-env": "^5.2.0",
-    "eslint": "5.16.0",
-    "eslint-config-standard": "^11.0.0",
-    "eslint-config-standard-react": "^6.0.0",
-    "eslint-plugin-import": "^2.13.0",
-    "eslint-plugin-node": "^7.0.1",
-    "eslint-plugin-promise": "^4.0.0",
-    "eslint-plugin-react": "^7.10.0",
-    "eslint-plugin-standard": "^3.1.0",
+    "eslint-config-prettier": "^6.11.0",
+    "eslint-plugin-prettier": "^3.1.4",
     "gh-pages": "^2.0.1",
+    "prettier": "^2.0.5",
     "react": "^16.9.0",
     "react-dom": "^16.13.1",
     "react-scripts": "^3.0.1",
```

Comment:
@arthurdenner this three lines its not in your changes, but.. `react`, `react-dom` and `react-script` makes sense in `devDependencies`? `react` already be in `peerDependencies`.

Reply:
Yes, they should be declared as `devDependencies`.